cmd/loadTests.go

```diff
@@ -1735,6 +1759,9 @@ func (h *ConcreteHandlerItsPipelines) validateItsPipeline(ctx *JourneyContext, a
 			return false, nil
 		}
 		if IntegrationTestsPipelineRun.IsDone() {
+			if !stage {
+				h.handlePVCSforITS(threadIndex, framework, IntegrationTestsPipelineRun)
```

Comment:
This function is a duplicate of handlePVCS function and it fails SonarCloud Duplicated Lines (%) on New Code
55.6% rule
The way to allow handlePVCSto be used for both handlers - ConcreteHandlerPipelines and ConcreteHandlerItsPipelines is - 

1. To change the below ConcreteHandlerItsPipelines struct definition 

type ConcreteHandlerItsPipelines struct {
	BaseHandler
}

To 
type ConcreteHandlerItsPipelines struct {
	*ConcreteHandlerPipelines // Embedding ConcreteHandlerPipelines
}

2 . Update the below 
h.handlePVCSforITS(threadIndex, framework, IntegrationTestsPipelineRun)

To 
h.handlePVCS(threadIndex, framework, IntegrationTestsPipelineRun)

3. Delete function handlePVCSforITS

Reply:
Updated the code :+1:

Reply:
Hi @siddardh-ra 
I found there is a change needed, I create a review